include/deal.II/fe/fe.h

```diff
@@ -149,6 +149,65 @@ template <int dim, int spacedim> class FESystem;
  * FiniteElement::system_to_block_index(). The number of blocks of a finite
  * element can be determined by FiniteElement::n_blocks().
  *
+ * To better illustrate these concepts, let's consider the following
+ * example
+ * @code
+ * FESystem<dim> fe_basis(FE_Q<dim>(2), dim, FE_Q<dim>(1),1);
+ * @endcode
+ * with <code>dim=2</code>. The resulting finite element has 3 components:
+ * two coming from quadratic element and one from linear element. In case
+ * of fluids, one can think of the first two components representing a
+ * vector-valued velocity field whereas the last one being a scalar valued
+ * pressure field. Without DoF renumbering this finite element will
+ * produce the following distribution of local degrees of freedom:
+ *
+ * @image html fe_system_example.png DoF indices
+ *
+ * Using the two functions FiniteElement::system_to_component_index() and
+ * FiniteElement::system_to_base_index() one can get the following info
+ * @code
+ * const unsigned int comp  = fe_basis.system_to_component_index(i).first;
+ * const unsigned int base  = fe_basis.system_to_component_index(i).second;
```

Comment:
I think `base` suggests that this is the base element. But it is really the shape function index *within this base*. Maybe call it `within_base`?